sdk/apimanagement/Microsoft.Azure.Management.ApiManagement/src/Microsoft.Azure.Management.ApiManagement.csproj

```diff
@@ -8,7 +8,7 @@
     <Description>Provides ApiManagement management capabilities for Microsoft Azure.</Description>
     <AssemblyTitle>Microsoft Azure API Management Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.ApiManagement</AssemblyName>
-    <Version>6.0.0-preview</Version>
+    <Version>7.0.0-preview</Version>
     <PackageTags>Microsoft Azure ApiManagement management;API Management;</PackageTags>
     <PackageReleaseNotes>Refer https://aka.ms/apimdotnetsdkchangelog for release notes.</PackageReleaseNotes>
```

Comment:
Is this AKA link still valid?  You should provide some text here, as people use this to decide whether to download the package or not.

Reply:
yes, this AKA will point to CHANGELOG.md

Reply:
@markcowl once this PR is merged, the aka.ms link will point to the correct changelog. I hope that is okay?

Reply:
@markcowl once this PR is merged, the aka.ms link will point to the correct changelog. I hope that is okay?